CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This release achieves 100% compliance with Python Array API specification (revis
 * Updated Python Array API specification version supported to `2024.12` [#2416](https://github.com/IntelPython/dpnp/pull/2416)
 * Removed `einsum_call` keyword from `dpnp.einsum_path` signature [#2421](https://github.com/IntelPython/dpnp/pull/2421)
 * Changed `"max dimensions"` to `None` in array API capabilities [#2432](https://github.com/IntelPython/dpnp/pull/2432)
+* Added CUDA compatibility for `dpnp.i0` and `dpnp.kaiser` functions [#2439](https://github.com/IntelPython/dpnp/pull/2439)
```

Comment:
It's more like a bug we fixed, because there was build error while building with CUDA enabled target (which is a legacy functionality now)